consensus/config/src/parameters.rs

```diff
@@ -97,6 +103,30 @@ impl Parameters {
         Duration::from_millis(500)
     }
 
+    pub(crate) fn default_max_blocks_per_fetch() -> usize {
+        if cfg!(msim) {
+            // Exercise hitting blocks per fetch limit.
+            10
+        } else {
+            1000
+        }
+    }
+
+    pub(crate) fn default_sync_last_known_own_block_timeout() -> Duration {
+        if cfg!(msim) {
+            Duration::from_millis(500)
+        } else {
+            // Here we prioritise liveness over the complete de-risking of block equivocation. 5 seconds
+            // in the majority of cases should be good enough for this given a healthy network.
+            Duration::from_secs(5)
+        }
+    }
+
+    pub(crate) fn default_propagation_delay_stop_proposal_threshold() -> u32 {
+        // In experiments, propagation delay is usually 0 round.
+        10
```

Comment:
I believe because of the streaming nature of TCP, this value is usually 0 in production. Occasionally it goes to 1. When a host has issues, usually it shoots up. Increased this to 20 to be on the safe side for the initial rollout.